src/tools/altinn-fleet-stats/backend/altinn_fleet/bpmn.py

```
@@ -0,0 +1,212 @@
+      "nodes": {
+        "<id>": {"kind": "task"|"gateway"|"startEvent"|"endEvent",
+                  "tag": "task", "altinn_type": "data"|"signing"|...,
+                  "name": "..."}
+      },
+      "edges": [{"id": "...", "source": "...", "target": "...", "name": "..."}, ...]
+    }
+    """
+    if not bpmn_path.exists():
+        return None
+    try:
+        tree = ET.parse(bpmn_path)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)